charts/player/charts/player-api/templates/configmap-seed.yaml

```diff
@@ -0,0 +1,213 @@
+{{- if eq .Values.identity.register true }}
+{{- $chartName := .Chart.Name }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "player-api.fullname" . }}-seed
+  labels:
+{{- include "player-api.labels" . | nindent 4 }}
+data:
+{{- range $key, $val := .Values.identity }}
+  {{ printf "%s-%s.json" $chartName $key }}: {{ $val | quote }}
+{{- end }}
+  seed.sh: |-
+    #!/bin/bash -xe
+    # 
+    # Copyright 2021 Carnegie Mellon University.
+    # Released under a BSD (SEI)-style license, please see LICENSE.md in the
+    # project root or contact permission@sei.cmu.edu for full terms.
+
+    #############################
+    #   Identity Seed Script    #
+    #############################
+
+    # Change to the current directory
+    DIRECTORY="$(dirname "$BASH_SOURCE[0]")"
+    cd $DIRECTORY
+    echo $DIRECTORY
+    CLIENT_NAME=${1:-}
+    CLIENT_FILE_NAME="${2:-$CLIENT_NAME}-client.json"
+    RESOURCE_NAME=${1:-}
+    RESOURCE_FILE_NAME="${3:-$CLIENT_NAME}-resource.json"
+    USER_FILE_NAME="${4:-$CLIENT_NAME}-user.json"
+    # http or https
+    PROTO="https"
+    DOMAIN="${DOMAIN:-foundry.local}"
+
+    echo "$RESOURCE_FILE_PATH"
+
+
+    # get access token
+    ACCESS_TOKEN=$(curl --silent --insecure --request POST \
+      --url "$PROTO://$DOMAIN/identity/connect/token" \
+      --data grant_type=password \
+      --data scope="identity-api identity-api-privileged" \
+      --data client_id=bootstrap-client \
+      --data client_secret=foundry \
+      --data username=administrator@foundry.local \
+      --data password=foundry | jq -r '.access_token')
```

Comment:
Hard-coded credentials here seems like it might be a problem? Could these be pulled from the values file or something?

Reply:
Good point. I'm not sure if we could use a values file, but we could use an environment variable.